src/foremast/templates/pipeline/pipeline_prod_datapipeline.json.j2

```diff
@@ -0,0 +1,5 @@
+[
+    {% include "pipeline/stage-judgement-nonprod.json.j2" %},
+    {% include "pipeline/stage-infrastructure-setup-datapipeline.json.j2" %}
+    {% endif %}
```

Comment:
oh, it does! I fixed it in our internal templates but forgot about this one. Good catch